roslibrust_codegen/src/integral_types.rs

```diff
@@ -36,6 +36,13 @@ impl RosMessageType for Time {
     const DEFINITION: &'static str = "";
 }
 
+// The equivalent of rospy AnyMsg or C++ ShapeShifter, subscribe_any() uses this type
+impl RosMessageType for Vec<u8> {
```

Comment:
I think we probably want to do a "newtype" pattern here, and say `type RawBytesMsg(Vec<u8>)` (ShapeShifter might be a better name for it).

And then `impl RosMessageType for RawBytesMsg` instead of for Vec<u8> directly?

Not a huge difference, but I am worried about what happens if a crate user calls `.subscribe::<Vec<u8>()` with this definition floating around? With the `newtype` pattern we can make sure "RawBytesMsg" is a private type of the crate and not available for the user to try to use somewhere else? Maybe doesn't matter?

Reply:
What you did is perfect!